Payments/Overview/02_Features.md

```diff
@@ -0,0 +1,44 @@
+# Key Features
+Organize and monitor your incoming payments and refunds:   
+
+- [Automated import](./02_Features.md#automated-import)      
```

Comment:
Diese Links sind schon im TOC, hier gehören sie also nicht wirklich hin. Hier schreiben wir in der Regel eine kurze zusammenfassung der Key features vom Modul. Es muss auch nicht super lang sein. Guck mal z.B. bei Omni-Channel oder Fulfillment (oder andere). Die Infos holen wir auch aus der Webseite, aber gehe mit der Info vorsichtig um, es ist nicht alles aktuell und/oder kann sich auf alte Module beziehen (z.B. hier auf Zahlungsabwicklung).